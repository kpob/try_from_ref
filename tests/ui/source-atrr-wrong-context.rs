#![no_main]

struct Source;

#[derive(derive_try_from::TryFromRef)]
#[source(Source)]
#[err(syn::Error)]
struct MyStruct {
    #[source(OtherSource)]
    field: u32
}