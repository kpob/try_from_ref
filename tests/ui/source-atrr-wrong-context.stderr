error: Attribute source used in the wrong context

         = note: Only source and err attributes are allowed here

 --> tests/ui/source-atrr-wrong-context.rs:9:5
  |
9 |     #[source(OtherSource)]
  |     ^^^^^^^^^^^^^^^^^^^^^^
